django/chat/models.py

```diff
@@ -192,7 +204,7 @@ def qa_prompt_combined(self):
         return ChatPromptTemplate(
             message_templates=[
                 ChatMessage(
-                    content=self.qa_system_prompt,
+                    content=current_time_prompt() + self.qa_system_prompt,
```

Comment:
Most chat tools e.g. ChatGPT, Bing, etc. seem to have this in their system prompt. It helps certain types of queries and only takes a few tokens.